lib/siwapp_web/templates/layout/nav_404.html.heex

```diff
@@ -0,0 +1,70 @@
+<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
```

Comment:
Navbar already existed. Whys is there a new navbar ?

Reply:
Because the other one asked for some assign impossible to obtain in this pipeline. So we needed to adjust a new one for this page

Reply:
Already there are socket version and conn version create a new one for just an error page ... if something change in the menu don't believe some remember that there is a specific nav for error view.

Conn nav can be adapted to work with error page ?